app/views/layouts/_navbar.html.haml

```diff
@@ -26,13 +26,12 @@
             = render partial: "layouts/nav/organizer_nav"
 
           - if current_event && policy(current_event).staff?
-            %li{class: "#{request.path == event_staff_path(current_event) ? 'active' : ''}"}
+            %li{class: "event-dashboard-link"}
               = link_to event_staff_path(current_event) do
                 %i.fa.fa-dashboard
                 %span Event Dashboard
 
-          %li{class: "#{request.path == notifications_path ? 'active' : ''}"}
-            = render partial: "layouts/nav/notifications_link"
+          = render partial: "layouts/nav/notifications_link"
```

Comment:
A stronger partial name might be notifications_menu or notifications_list vs link as the content of the partial is more about that vs a nav bar item.